koku/sources/storage.py

```diff
@@ -26,17 +26,46 @@ class SourcesStorageError(Exception):
     """Sources Storage error."""
 
 
```

Comment:
It looks like this `storage.py` file could be getting a bit long. You might want to consider breaking it up into per-provider files.